### PR TITLE
Fix weapon muzzle texture cutoff

### PR DESCRIPTION
**Pull request by Jupeyy**
As reported by Pipou on discord, the weapon muzzle is cut off, bcs of wrong values in content.py,

this

changing them would result in a scale(it would look different), so i hardcoded a few values to let it look the same

new:
screenshot_2020-10-25_14-15-08

old:
screenshot_2020-10-24_22-56-03

Tho i'd like to think about it, or maybe someone else thinks about it xd

### CODE REVIEW

File: datasrc/content.py [hunk 1]
```diff
@@ -498,7 +498,7 @@ def FileList(format, num):
 weapon.visual_size.Set(64)
 weapon.offsetx.Set(32)
 weapon.offsety.Set(4)
-weapon.muzzleoffsetx.Set(50)
+weapon.muzzleoffsetx.Set(50 + 8.8752)
```
Comment (def-) [inline datasrc/content.py:501]: please add a comment explaining the magic number

Comment (Jupeyy) [reply datasrc/content.py:501]: not so ez, have to explain half of vanillas strange sprite calculations xdd, but yeah will do

**Comment by Jupeyy** [comment]
hope this helps @def-

**Comment by def-** [comment]
bors r+
Thanks

**Pull request merged**